### PR TITLE
smp v2 llama2 training example using fp8

### PR DESCRIPTION
**Pull request by aruncs2005**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: 3.test_cases/17.SM-modelparallelv2/conda_env_setup.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+conda install pytorch="2.2.0=sm_py3.10_cuda12.1_cudnn8.9.5_nccl_pt_2.2_tsm_2.2_cuda12.1_0" packaging --override-channels \
+  -c https://sagemaker-distributed-model-parallel.s3.us-west-2.amazonaws.com/smp-v2/ \
+  -c pytorch -c numba/label/dev \
+  -c pytorch-nightly -c nvidia -c conda-forge
+
+# Install dependencies of the script as below
+
+python -m pip install --no-cache-dir -U \
+    "transformers==4.37.1" \
```
Comment (goswamig) [inline 3.test_cases/17.SM-modelparallelv2/conda_env_setup.sh:35]: Some of these dependencies are specific to SMP-v2, do. we need to clearly specify here?

Comment (aruncs2005) [reply 3.test_cases/17.SM-modelparallelv2/conda_env_setup.sh:35]: sure we can do that.

**Review by goswamig — APPROVED** [review]
LGTM

**Review by awsankur — APPROVED** [review]
Looks good to me

**Pull request merged**